doc/cephfs/fuse.rst

```diff
@@ -1,54 +1,106 @@
-	sudo scp {user}@{server-machine}:/etc/ceph/ceph.conf /etc/ceph/ceph.conf
+    # on client host
+    mkdir /etc/ceph
+    ssh {user}@{mon-host} "sudo ceph config generate-minimal-conf" | sudo tee /etc/ceph/ceph.conf
 
-#. From your client host, copy the Ceph keyring from the monitor host to 
-   to the ``/etc/ceph`` directory. :: 
+   Alternatively, you may copy the conf file. But the above method creates a
+   conf with minimum details which is better.
 
-	sudo scp {user}@{server-machine}:/etc/ceph/ceph.keyring /etc/ceph/ceph.keyring
+#. Ensure that the conf has appropriate permissions::
 
-#. Ensure that the Ceph configuration file and the keyring have appropriate 
-   permissions set on your client machine  (e.g., ``chmod 644``).
+    chmod 644 /etc/ceph/ceph.conf
 
-For additional details on ``cephx`` configuration, see 
-`CEPHX Config Reference`_.
+#. Create the CephX user and get its secret key::
 
-To mount the Ceph file system as a FUSE, you may use the ``ceph-fuse`` command.
-For example::
+    ssh {user}@{mon-host} "sudo ceph fs authorize cephfs client.foo / rw" | sudo tee /etc/ceph/ceph.client.foo.keyring
 
-	sudo mkdir /home/username/cephfs
-	sudo ceph-fuse -m 192.168.0.1:6789 /home/username/cephfs
+   In above command, replace ``cephfs`` with the name of your CephFS, ``foo``
+   by the name you want for your CephX user and ``/`` by the path within your
+   CephFS for which you want to allow access to the client host and ``rw``
+   stands for both read and write permissions. Alternatively, you may copy the
+   Ceph keyring from the MON host to client host at ``/etc/ceph`` but creating
+   a keyring specific to the client host is better.
 
-If you have more than one file system, specify which one to mount using
-the ``--client_mds_namespace`` command line argument, or add a
-``client_mds_namespace`` setting to your ``ceph.conf``.
+.. note:: In case you are deploying Ceph in a non-production scenario you
+   may also use admin's secret key instead.
 
-See `ceph-fuse`_ for additional details.
+.. note:: If you get 2 prompts for password while running above any of 2 above
+   command, run ``sudo ls`` (or any other trivial command with sudo)
+   immediately before these commands.
 
-To automate mounting ceph-fuse, you may add an entry to the system fstab_.
-Additionally, ``ceph-fuse@.service`` and ``ceph-fuse.target`` systemd units are
-available. As usual, these unit files declare the default dependencies and
-recommended execution context for ``ceph-fuse``. An example ceph-fuse mount on
-``/mnt`` would be::
+#. Ensure that the keyring has appropriate permissions::
 
-	sudo systemctl start ceph-fuse@/mnt.service
+    chmod 600 /etc/ceph/foo.keyring
 
-A persistent mount point can be setup via::
+Synopsis
+--------
+In general, the command to mount CephFS via FUSE looks like this::
 
-	sudo systemctl enable ceph-fuse@/mnt.service
+    ceph-fuse {mountpoint} {options}
 
-For troubleshooting, see :ref:`ceph_fuse_debugging`.
+Mounting CephFS
```

Comment:
FYI: https://tracker.ceph.com/issues/42601

Reply:
Referred it in one of my commit message.